tasks/openshift_deploy.yml

```diff
@@ -1,6 +1,17 @@
-- name: Deploying container {{ item }} into OpenShift 4.3 environment
-  block:
-  - name: Deploy {{ item }}
+- block:
+  - name: Pull image {{ stuff.registry_redhat_io }} from registry.redhat.io
```

Comment:
I pulled down these changes and was trying to run them against my OpenShift cluster. It fails for me on this task since docker is not available on my RHEL 8.1 machine. I had to remove this task in order for it to continue on.

I think this task can be removed as the following task issuing `oc import-image` is telling the client to get the image from the registry defined.

Console output
```
2020-08-18 09:23:45,783 INFO TASK [debug] **************************************************************************************************************************************************
2020-08-18 09:23:45,808 INFO ok: [10.0.102.240] => {
2020-08-18 09:23:45,808 INFO "oc_import": {
2020-08-18 09:23:45,809 INFO "changed": true,
2020-08-18 09:23:45,809 INFO "cmd": "oc import-image perl:5.26 --from=registry.redhat.io/rhel8/perl-526:latest --insecure=true --confirm",
2020-08-18 09:23:45,809 INFO "delta": "0:00:01.359698",
2020-08-18 09:23:45,809 INFO "end": "2020-08-18 09:23:44.644262",
2020-08-18 09:23:45,809 INFO "failed": false,
2020-08-18 09:23:45,809 INFO "rc": 0,
```

Reply:
Ok, I have discussed it with OpenShift colleagure and docker pull is not needed.